operator/generate.go

```diff
@@ -0,0 +1,6 @@
+//go:build generate
+
+// Generate manifests
+//go:generate go run -tags generate sigs.k8s.io/controller-tools/cmd/controller-gen rbac:roleName=manager-role paths=./... output:artifacts:config=../package/rbac
```

Comment:
Note for future packaging with crossplane: I had to change the path where the RBAC rules are created. See https://github.com/vshn/provider-cloudscale/pull/23
```suggestion
//go:generate go run -tags generate sigs.k8s.io/controller-tools/cmd/controller-gen rbac:roleName=manager-role paths=./... output:artifacts:config=../.kind/rbac
```